pytorch_lightning/tuner/lr_finder.py

```diff
@@ -311,7 +312,7 @@ def func():
 
         return func
 
-    def plot(self, suggest: bool = False, show: bool = False):
+    def plot(self, suggest: bool = False, show: bool = False) -> 'plt.Figure':
```

Comment:
'plt.Figure' is causing PEP8 to fail. `./pytorch_lightning/tuner/lr_finder.py:315: [F821] undefined name 'plt'`
What should be done here? @justusschock

Reply:
is pyplot imported there?

Reply:
pyplot is imported inside the function